checks/linux.py

```diff
@@ -51,8 +51,14 @@ def checkDistro(lines):
     distro = isDistroNix[0].split()
     distro = distro[2:]
     distro = ' '.join(distro)
+    distroHelp = ''
 
-    return [LEVEL_INFO, distro, ""]
+    # this is logged when the file can't be found at all
+    if distro == 'Missing /etc/os-release !':
+        distro = '(Missing)'
+        distroHelp = 'No distribution detected. This can lead to undefined behaviour. Please consult your distribution\'s support channels on how to fix this.<br>'
```

Comment:
```suggestion
        distroHelp = 'No distribution detected. This can lead to undefined behavior. Please consult your distribution\'s support channels on how to fix this.<br>'
```

Reply:
Changed the other 2 occurrences too or it would look weird.